src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -748,12 +740,13 @@ FixedwingAttitudeControl::task_main()
 	vehicle_status_poll();
 	vehicle_land_detected_poll();
 	battery_status_poll();
+	_sub_airspeed.update();
```

Comment:
Yes it's ugly. Hoping to encourage someone to move the rest towards the newer APIs.